### PR TITLE
dev, v3.0/reference/tools: update TiDB Lightning deployment document

### PR DESCRIPTION
**Pull request by lance6716**
Update Lightning config to latest.

### What is changed, added or deleted? 
Update Lightning configure file to latest version.

### Which version does your change affect? 
v3.0

### Checklist 

- [ ] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 1]
```diff
@@ -258,7 +258,7 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
 
 2. 将数据源写入到同样的机器。
 
-3. 配置 `tidb-lightning.toml`。
+3. 配置 `tidb-lightning.toml`。对于没有出现在下述模版中的配置，TiDB-Lightning 给出配置错误的提醒并退出。
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
3. 配置 `tidb-lightning.toml`。对于没有出现在下述模版中的配置，TiDB Lightning 给出配置错误的提醒并退出。
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 2]
```diff
@@ -380,6 +388,8 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
     pd-addr = "172.16.31.4:2379"
     # tidb-lightning 引用了 TiDB 库，而它自己会产生一些日志。此设置控制 TiDB 库的日志等级。
     log-level = "error"
+    # MySQL SQL Mode 配置
+    #sql-mode = ""
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    # sql-mode = ""
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 3]
```diff
@@ -347,6 +353,8 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
     #  - binary：不尝试转换编码
     # 注意，此参数不影响 Lightning 读取数据文件。
     character-set = "auto"
+    # 是否区分大小写
+    #case-sensitive = false
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    # case-sensitive = false
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 4]
```diff
@@ -289,9 +289,15 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
...
     max-size = 128 # MB
-    max-days = 28
+    max-days = 28 # 默认值为不删除旧日志
     max-backups = 14
 
+    # Server 模式
+    # 是否启用 Server 模式
+    #server-mode = false
+    # Server 模式下的监听地址
+    #status-addr = ":8289"
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    # status-addr = ":8289"
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 5]
```diff
@@ -289,9 +289,15 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
     level = "info"
     file = "tidb-lightning.log"
     max-size = 128 # MB
-    max-days = 28
+    max-days = 28 # 默认值为不删除旧日志
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]: Do you mean “默认不删除旧日志” ？

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 6]
```diff
@@ -312,7 +318,7 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
     #dsn = "/tmp/tidb_lightning_checkpoint.pb"
     # 导入成功后是否保留断点。默认为删除。
     # 保留断点可用于调试，但有可能泄漏数据源的元数据。
-    # keep-after-success = false
+    #keep-after-success = false
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    # keep-after-success = false
```

File: v3.0/reference/tools/tidb-lightning/deployment.md [hunk 7]
```diff
@@ -289,9 +289,15 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/how-to/deploy/orche
     level = "info"
     file = "tidb-lightning.log"
     max-size = 128 # MB
-    max-days = 28
+    max-days = 28 # 默认值为不删除旧日志
     max-backups = 14
 
+    # Server 模式
+    # 是否启用 Server 模式
+    #server-mode = false
```
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    # server-mode = false
```

File: v3.0/reference/tools/tidb-lightning/deployment.md (hunk 2 above)
Comment (lance6716) [reply v3.0/reference/tools/tidb-lightning/deployment.md]: I followed previous comment style of L271, L282, etc, which doesn't have a seperating space on commented config lines.

File: v3.0/reference/tools/tidb-lightning/deployment.md (hunk 5 above)
Comment (lance6716) [reply v3.0/reference/tools/tidb-lightning/deployment.md]: Yes, that's more fluent.

**Comment by lance6716** [comment]
This update affect `dev`, I'll update that version based on this edition.

File: v3.0/reference/tools/tidb-lightning/deployment.md (hunk 2 above)
Comment (TomShawn) [reply v3.0/reference/tools/tidb-lightning/deployment.md]: That's not the correct format (very likely done by mistake). Please also change them and make sure that no such format is in the document, thanks!

File: v3.0/reference/tools/tidb-lightning/deployment.md (hunk 5 above)
Comment (TomShawn) [inline v3.0/reference/tools/tidb-lightning/deployment.md]:
```suggestion
    max-days = 28 # 默认不删除旧日志
```

**Comment by lance6716** [comment]
PTAL @TomShawn

**Pull request merged**